software/atomic-ai-calendar-assistant

```diff
@@ -0,0 +1,20 @@
+name: "Atomic - an AI calendar assistant"
+website_url: "https://atomiclife.app"
+source_code_url: "https://github.com/rush86999/atomic"
+description: "Atomic, allows flexible timeblocking with priority, flexible meetings and generating new breaks and reminders powered by ChatGPT. It runs on Autopilot and is adjusted as needed. You can teach your time preferences and priority for recurring tasks and use automatted tagging to modify behavior as well."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Kotlin
+  - Python
+  - Nodejs
+  - Docker
+  - Typescript
+  - Yaml
+tags:
+  - Automation
+  - Calendar & Contacts - CalDAV or CardDAV Servers
+  - Booking & Scheduling
+  - Task Management & To-do Lists
```

Comment:
```suggestion
tags:
  - Calendar & Contacts - CalDAV or CardDAV Servers
  - Task Management & To-do Lists
```

Reply:
It has priority aka handshake links, which are like Calendly links but only generated from the calendar owner for a limited time window.

Reply:
sorry, we can just go with your suggestion. I can't seem to press 'commit suggestion' any longer.

Reply:
This needs to change anyway with the commit of #196